### PR TITLE
Fixes #24204: Hooks page looks broken since bootstrap 5 update

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/24204

Fix the display and the scrollspy of the Hooks page :
hooks

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
PR rebased

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/policyServerManagement.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h1>
 
       <div class="template-main">
         <div class="main-container">
-          <div class="main-details" data-bs-spy="scroll" data-bs-target="#navbar-scrollspy">
+          <div class="main-details" data-bs-spy="scroll" data-bs-target="#navbar-scrollspy" data-bs-smooth-scroll="true">
```
Comment (clarktsiory) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/policyServerManagement.html:51]: :+1:

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by clarktsiory** [comment]
OK, merging this PR

**Comment by clarktsiory** [comment]
OK, merging this PR

**Pull request merged**